src/lib/util.js

```diff
@@ -205,6 +205,10 @@ function throttle(fn, time, context) {
   return wrapperFn;
 }
 
+module.exports.toDenseArray = function(x) {
```

Comment:
while small, lets put this in its own file. I want to break `lib/util` up a bunch.